libbeat/cmd/instance/beat.go

```diff
@@ -345,7 +345,14 @@ func (b *Beat) createBeater(bt beat.Creator) (beat.Beater, error) {
 
 	reg := b.Monitoring.StatsRegistry().GetOrCreateRegistry("libbeat")
 
-	err = metricreport.SetupMetrics(b.Info.Logger.Named("metrics"), b.Info.Beat, version.GetDefaultVersion()) //nolint:staticcheck //keep behavior for now
+	err = metricreport.SetupMetricsOptions(metricreport.MetricOptions{
+		Name:           b.Info.Beat,
+		Version:        version.GetDefaultVersion(),
+		EphemeralID:    b.Info.ID.String(),
```

Comment:
@nodece , `b.Info.ID.String()` is different from `EphemeralID`. The former is an identifier for the host, the latter an identifier for the current run. Could you restore `metricreport.EphemeralID()`?

I know it's deprecated, but we can deal with it later.
Also could you refrain for force pushing? it makes it harder to see the latest changes.